app/app.js

```diff
@@ -63,6 +63,29 @@ function (app, $, _, Backbone, Bootstrap, Helpers, Utils, FauxtonAPI, Couchdb) {
   // Localize or create a new JavaScript Template object
   var JST = window.JST = window.JST || {};
 
+  var parseCookies = function (cookies) {
+    var list = {}, rc = cookies;
+    rc && rc.split(';').forEach(function (cookie) {
```

Comment:
Can you rather change the `rc` check to an if statement. Its better to make it a bit more explicit.